src/AdvancedListingBlockTemplate.jsx

```diff
@@ -46,16 +50,58 @@ const AdvancedListingBlockTemplate = ({
   } else if (headerHref) {
     moreLink = <a href={headerHref}>{moreLinkText || headerHref}</a>;
   }
+  const getEventDate = (item) => {
+    let start = '',
+      end = '';
+
+    if (item.start) {
+      const parsedDate = new Date(Date.parse(item.start));
+      start = `${parsedDate.toLocaleString('default', {
+        month: 'short',
+      })} ${parsedDate.getDate()}, ${parsedDate.getFullYear()}`;
+    }
+
+    if (item.end) {
+      const parsedDate = new Date(Date.parse(item.end));
+      end = `${parsedDate.toLocaleString('default', {
+        month: 'short',
+      })} ${parsedDate.getDate()}, ${parsedDate.getFullYear()}`;
+    }
+    if (end == start) {return start}
+    else {return start + ' - ' + end;};
+  };
+  const getEventTime = (item) => {
+    let start = '',
+      end = '';
 
+    if (item.start) {
+      const parsedDate = new Date(Date.parse(item.start));
+      start = `${parsedDate.toLocaleString(
+        'en-US',
+        {hour: 'numeric', minute: 'numeric', hour12: true},
+      )}`;
```

Comment:
This should be done in some generic way, not to be attached to en-US locale

Reply:
changed locales to 'default' instead of 'en-US'